# Algortihm Parameters
alpha: 0.1
gamma: 0.8
epsilon: 0.9
epsilon_discount: 0.999 # 1098 eps to reach 0.1
nepisodes: 8
nsteps: 100

# Environment Parameters
desired_pose:
    x: 5.0
    y: 0.0
    z: 0.0
max_altitude: 2.0   # in meters
max_incl: 0.7       # in rads
speed_value: 1.0    # in m/s
running_step: 0.3   # in seconds
